.github/workflows/scorecard.yml

```diff
@@ -31,16 +31,13 @@ jobs:
       # actions: read
 
     steps:
-      - name: Setup Node.js environment
-        uses: actions/setup-node@v4
-
       - name: "Checkout code"
-        uses: actions/checkout@v4
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
```

Comment:
Because this is what Bazel is using, so I made it match.

A different answer is that the scorecard people recommend pinning to checksums, so that it is harder for someone to hijack the tag to point to something else. But, they also provide examples which just use tags, so their advice is inconsistent.

Reply:
Since this is a commonly used action released on GH marketplace (https://github.com/marketplace/actions/checkout), I think it would be fine to use the tag? It's more clear, and we should probably also update the yml file in Bazel.

Reply:
For what it is worth, I only use the checksum format for repositories where I manage the dependencies using Renovate. The other repositories use the tag.

Reply:
I'm going to resolve and merge just to get us back running.

@meteorcloudy I'm leaving it up to you to resolve this as a bazelbuild repository policy. When that gets figured out and changed for Bazel, we can make this match.

Reply:
OK, it looks like the dependabot is able to update this hash: https://github.com/bazelbuild/bazel/commit/5125cddf55f0c66f86a876c96c485e51ea9cd331#diff-2e3112f4e81a9c47df8000638ce3b1b9ca15edcc82b228c207a7a4ff3bc7133fL40

So I guess using a commit hash is fine.